vue/BUILD.bazel

```diff
@@ -1,6 +1,12 @@
 load("@npm//:defs.bzl", "npm_link_all_packages")
 load("@npm//vite:package_json.bzl", vite_bin = "bin")
 load("@npm//vue-tsc:package_json.bzl", vue_tsc_bin = "bin")
+load("@aspect_rules_js//npm:defs.bzl", "npm_link_package")
+ 
+npm_link_package(
+    name = "node_modules/@yourname/yourlibrary",
```

Comment:
Is this used?

Reply:
yes, it's used below on line 25, along with all the 3p packages